python/cuspatial/benchmarks/api/bench_api.py

```diff
@@ -92,26 +92,20 @@ def bench_sinusoidal_projection(benchmark, gpu_dataframe):
 
 
 def bench_directed_hausdorff_distance(benchmark, sorted_trajectories):
-    benchmark(
-        cuspatial.directed_hausdorff_distance,
-        sorted_trajectories[0]["x"],
-        sorted_trajectories[0]["y"],
-        sorted_trajectories[1],
-    )
+    coords = sorted_trajectories[0][["x", "y"]].interleave_columns()
+    offsets = sorted_trajectories[1]
+    s = cuspatial.GeoSeries.from_multipoints_xy(coords, offsets)
+    benchmark(cuspatial.directed_hausdorff_distance, s)
```

Comment:
These came from https://github.com/rapidsai/cuspatial/pull/924 and probably will not run until we merge that one.